integrations/spring-boot/pom.xml

```diff
@@ -23,6 +23,23 @@
             <version>${org.springframework.boot.version}</version>
             <scope>provided</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web-services</artifactId>
+            <version>${org.springframework.boot.version}</version>
```

Comment:
Why we need a spring web-services in this module?

Reply:
It was a left-over from an earlier version with more functionality that required it. Removed.